geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1694,11 +1698,19 @@ def call(fnames, command_line_args=None):
 
             for output_product in final_products[cpath]["files"]:
                 output_checker = output_checkers.get_plugin(output_product)
-                curr_retval = output_checker(
-                    output_checker,
-                    cpath,
-                    [output_product],
-                )
+                if output_checker.name == "image":
+                    curr_retval = output_checker(
+                        output_checker,
+                        cpath,
+                        [output_product],
+                        image_compare_threshold,
+                    )
+                else:
+                    curr_retval = output_checker(
+                        output_checker,
+                        cpath,
+                        [output_product],
+                    )
```

Comment:
Again, passing in output_checker_kwargs could remove the "image" conditional - just pass kwargs through directly.